include/pcmki/pcmki_simulate.h

```diff
@@ -13,6 +13,17 @@
 #include <crm/pengine/pe_types.h>
 #include <stdbool.h>
 
+typedef enum {
+    sim_none             = 0,
+    sim_all_actions      = 1 << 0,
+    sim_show_pending     = 1 << 1,
+    sim_process          = 1 << 2,
+    sim_show_scores      = 1 << 3,
+    sim_show_utilization = 1 << 4,
+    sim_simulate         = 1 << 5,
+    sim_store            = 1 << 6,
+} pcmk__sim_flags;
```

Comment:
I thought about doing that and then didn't.  I should have just gone for it.  Because we're going to want the flags to be public, I assume there should be only a single underscore on all those?  I'd additionally want to define them in include/pacemaker.h instead, right?  I think all the include/pcmki/ headers are supposed to be internal.

Reply:
Correct on all counts

Reply:
Fixed.